### DISCUSSION TITLE
Watch List

### DISCUSSION THREAD

**Discussion by OddHouseGames** [community/community#3576] [Mobile]
Unless I'm mistaken, there is no way to view a list of everything you are watching? 

You would need to wait for alerts to come in and then visit the project and toggle as needed?

It would be great to have a watch list, similar to how you can view starred repositories.

**Comment by candyho** [comment]
Thanks your your feedback! I believe the notification tab is the closest to what you are looking for. Can you elaborate more on the cases where you find the notification tab cannot fulfil your needs for all the "watch" activities?

**Reply by OddHouseGames** [reply]
Hi there, I may just be an edge case, but occasionally I audit the repositories that I'm following, but when you click on user profile, you don't see a list of 'watching' so you can't batch audit your list of watches, you just need to keep an eye on notifications and then over time based on alerts that come in, you can turn it off.

**Reply by OddHouseGames** [reply]
It feels like something could be visibility on this page, but perhaps it wouldn't be useful enough for general users although there is lots of space on the profile page (screenshot shown)

![image](https://user-images.githubusercontent.com/62911518/112367379-366a4900-8cb0-11eb-8a22-04fbf3c9281c.jpeg)

**Comment by p292** [comment]
Resurrecting from the dead … I was surprised to find this feature isn’t enabled. In short it’s just an iOS way to get to GitHub.com/watching
